server/tinode.conf

```diff
@@ -325,6 +325,9 @@
 				// Password to use when authenticating the sender; used only if "login" is provided.
 				"sender_password": "your-password-here",
 
+				// Mechanism supported for authentication we can use plain, login or md5
```

Comment:
`// Authentication mechanism to use, optional. One of "login", "md5", "plain" (default).`